<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>351</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>247</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QLineEdit" name="lineEditLogin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>jean</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>123</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnexion">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLoginState">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogin_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../wish-o-removebg-preview.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRegister">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>S'inscrire</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPseudo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asgarrrr</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPseudo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Pseudo</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonConnexion</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>oauth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditLogin</sender>
   <signal>returnPressed()</signal>
   <receiver>Login</receiver>
   <slot>oauth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditPassword</sender>
   <signal>returnPressed()</signal>
   <receiver>Login</receiver>
   <slot>oauth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRegister</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>switchState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>oauth()</slot>
  <slot>regist()</slot>
  <slot>switchState()</slot>
 </slots>
</ui>
